### PR TITLE
Background Image Capture added as Immediate Capture with quick Actions

### PR DESCRIPTION
**Pull request by SamarpanCoder2002**
Related Issue: #56 

- Added Background Capture Image Facility with quick Actions

Closes: #56

### Description

I have added a background image capture facility with quick actions.... Please test it by connecting the physical device with your IDE(Or build the apk and try on a physical device to install it)...because Emulator may not work properly for quick actions or background service due to not having a camera)

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behavior of your code (Write NA if not)
Testing on the Emulator may not open the camera....Please try the connecting physical device(Activating Developer Options) with your IDE or Make an APK file and install it on your phone to test it...

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.

## Additional Info (optional)
@shagun25 @hiashutoshsingh please review it and merge it.... If any problem you can freely tell me.... I am also a GSSOC'21 participant...Thank u @hiashutoshsingh @shagun25 

Video Demo: https://drive.google.com/file/d/1XEGdAWkbbZINLCQzPZZJ04QKMn30nAv2/view?usp=sharing

### CODE REVIEW

**Comment by SamarpanCoder2002** [comment]
> Please take a look here @shagun25 @hiashutoshsingh
@shagun25 @hiashutoshsingh if there is no prb, please merge the pull request....For any problem, you can freely tell me... Actually, I can't make another pull request until it gets merged.... Thank u

**Review by hiashutoshsingh — CHANGES_REQUESTED** [review]
Please resolve the conflict also.

File: lib/services/calls_and_messages_service.dart [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 class CallsAndMessagesService {
   void call(String number) => launch("tel://$number");
 
   void sendSms(String number) => launch("sms:$number");
 
   void sendEmail(String email) => launch("mailto:$email");
+
+  void takePhoto() async {
+    print("Take Image");
+    ImagePicker.pickImage(source: ImageSource.camera)
```
[the remark this answers is not in the document]
Comment (SamarpanCoder2002) [reply lib/services/calls_and_messages_service.dart:16]: Open the camera to take image(Try in physical device as mentioned above)

File: lib/main.dart [hunk 2]
```diff
@@ -68,9 +67,13 @@ class _MyAppState extends State<MyApp> {
           type: 'action_two',
           localizedTitle: 'Emergency Call',
           icon: 'AppIcon'),
-    ]).then((value) {
-    });
-    _volumeButtonSubscription = HardwareButtons.volumeButtonEvents.listen((event) {
+      const ShortcutItem(
+          type: 'action_three',
+          localizedTitle: 'Immediate Capture',
```
[the remark this answers is not in the document]
Comment (SamarpanCoder2002) [reply lib/main.dart:72]: As like Emergency call or message, Image Capture added with quick actions

[the remark this answers is not in the document]
Comment (SamarpanCoder2002) [reply lib/main.dart:72]: Like that Immediate Capture as quick Actions

File: lib/services/calls_and_messages_service.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (SamarpanCoder2002) [reply lib/services/calls_and_messages_service.dart:16]: ImageSource.camera open the camera to take the image.....please try it as mentioned above.... I also attached a video link...Take a look here..... @shagun25 @hiashutoshsingh

**Comment by SamarpanCoder2002** [comment]
> Please resolve the conflict also.

@shagun25 @hiashutoshsingh ....Conflicts Solved and and all questions answered..... Hope you review it and merge it soon @hiashutoshsingh

File: lib/services/calls_and_messages_service.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiashutoshsingh) [reply lib/services/calls_and_messages_service.dart:16]: Perfect!

**Review by hiashutoshsingh — APPROVED** [review]
Thanks for the contribution.

**Pull request merged**